app/assets/v2/css/activity_stream.css

```diff
@@ -79,6 +79,10 @@ a .sup {
 .action.open {
   background-color: #eeeeee;
 }
+.emoji-container,
```

Comment:
^ could you add an empty line between CSS rules
(feel free if you've got the patience to do it for this css file

Reply:
On the new commit, I have added an empty line and space between curly bracket for all the css rules for a more consistent code style